<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CNH57O0" failures="1" tests="1" name="Error Validation Test" time="20.432" errors="0" timestamp="2025-05-14T00:10:20 WEST">
  <testcase classname="MohamedHafdi.Tests.ErrorValidation" name="logingErrorValidation" time="2.257">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at MohamedHafdi.Tests.ErrorValidation.logingErrorValidation(ErrorValidation.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- logingErrorValidation -->
</testsuite> <!-- Error Validation Test -->
